src/components/application_manager/src/commands/mobile/get_way_points_request.cc

```diff
@@ -32,18 +39,102 @@ void GetWayPointsRequest::Run() {
                  true);
 }
 
+bool GetWayPointsRequest::CheckResponseSyntax(
+    const smart_objects::SmartObject& obj) {
+  const char* str = obj.asCharArray();
+  const bool is_str_empty = str && !str[0];
+  return is_str_empty || CheckSyntax(str);
+}
+
+bool GetWayPointsRequest::ValidateSmartMapResponse(
+    const smart_objects::SmartObject& obj, ConstSetIter key) {
```

Comment:
@orhan-mehmedov , in this case, I think there is no difference, we have a copy of all keys